02/Form.js

```diff
@@ -1,10 +1,21 @@
 import React from 'react';
 
 class Form extends React.Component {
+
+    secRef = React.createRef()
+
+    handleSubmit = (e) => {
+        e.preventDefault();
+        const username = this.secRef.current.value;
+        this.props.handleSubmit(username);
```

Comment:
Warto też usuwać zawartość inputa po dodaniu tj. `this.secRef.current.value = '';`